packages/frontend/src/components/UserSettings/ProjectManagementPanel/index.tsx

```diff
@@ -38,9 +30,7 @@ const ProjectListItem: FC<{
 }> = ({ isCurrentProject, project: { projectUuid, name, type } }) => {
     const { user } = useApp();
     const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
-    const { mutate: deleteProjectMutation, isLoading: isDeleting } =
-        useDeleteProjectMutation();
-    const { mutate: deleteLastProjectMutation } =
```

Comment:
I can't find any reference to this in the new modal. we need to call this in order to remove it from the localStorage if it's the active project that user is deleting

Reply:
It is done here, in this file, like before, in a callback  instead of the modal . Let me know if you want me to change this. https://github.com/lightdash/lightdash/pull/5709/files#diff-718ea4de3540be10337e33184af77ff0d804d58f7be84651dc81d0aba63eab7eR86

Reply:
there are two mutations that needs to be run 

this is old code and it's very confusing. perhaps deleteProjectMutation should call that `deleteLastProjectMutation` mutation. one deletes the project and another deletes localStorage entry if it's an active project
```
                            onClick={() => {
                                deleteProjectMutation(projectUuid);
                                if (isCurrentProject) {
                                    deleteLastProjectMutation();
                                }
                            }}
```